src/Events/Custom_Tables/V1/WP_Query/Modifiers/Events_Series_Relationship_Modifier.php

```diff
@@ -23,24 +22,7 @@
  *
  * @package TEC\Events\Custom_Tables\V1\WP_Query\Modifiers
  */
-class Events_Series_Relationship_Modifier extends Base_Modifier {
-	use With_Series_Normalization;
-
-	/**
-	 * {@inheritdoc}
-	 */
-	public function hook() {
-		add_filter( 'posts_join', [ $this, 'join_on_series_relationships_table' ], 10, 2 );
-		add_filter( 'posts_where', [ $this, 'where_event_is_related_to_series' ], 10, 2 );
-	}
-
-	/**
-	 * {@inheritdoc}
-	 */
-	public function unhook() {
-		remove_filter( 'posts_join', [ $this, 'join_on_series_relationships_table' ] );
-		remove_filter( 'posts_where', [ $this, 'where_event_is_related_to_series' ] );
-	}
+class Events_Series_Relationship_Modifier extends Occurrences_Series_Relationship_Modifier {
```

Comment:
Should this class be here at all? Series relationships and its effects are not a concern that should be handled in TEC at all.
It was erroneously here before, but this looks like an excellent chance to move it to ECP for good.